secure-ios-app/accesscontrol/AccessControlViewController.swift

```diff
@@ -49,9 +50,9 @@ class AccessControlViewController: UIViewController, UITableViewDataSource, UITa
         // Dispose of any resources that can be recreated.
     }
     
-    func highlightUserRealmRoles(identity: Identity) {
+    func highlightUserRealmRoles(user: User) {
```

Comment:
@twnolan Also possible to have asciidoc tags for this one too? `highlightUserRealmRoles`